Source/Engine/Core/Math/Vector2.cs

```diff
@@ -301,6 +301,19 @@ public void Normalize()
             }
         }
 
+        /// <summary>
+        /// Gets the normalized vector. Returned vector has length equal 1.
+        /// </summary>
+        public Vector2 Normalized
+        {
+            get
+            {
+                Vector2 result = this;
+                result.Normalize();
+                return result;
```

Comment:
could this just be simplified to `return this.Normalize()`?

Reply:
That would simplify things, thanks I'll adjust

Reply:
No,` Normalize()` returns a void

Reply:
Ah, okay didn't know that. Makes sense then. Could something like this also be useful for vector3?

Reply:
It already exists, but I found it strange that the same function did not exist in Vector2 so I created it. I just copied the same function from Vector3 to Vector2